docs/exterminate.rst

```diff
@@ -37,14 +33,30 @@ Examples
     Kill the selected unit.
 ``exterminate``
     List the targets on your map.
-``exterminate BIRD_RAVEN:male``
+``exterminate BIRD_RAVEN:MALE``
     Kill the ravens flying around the map (but only the male ones).
-``exterminate undead magma``
-    Kill all undead on the map by pouring magma on them.
+``exterminate GOBLIN --method MAGMA --only-visible --only-hostile``
+    Kill all visible, hostile goblins on the map by drowning them in magma.
+
+Options
+-------
+
+``--method <method>``
+    Specifies the "method" of killing units. See ``exterminate --help`` for a
```

Comment:
if they're reading this, they're already reading the output of `exterminate --help`. that part might not be helpful to say here.